### ISSUE TITLE
Cattrs lib issue

### ISSUE THREAD

**Issue by Raghuveer123** [python-attrs/cattrs#468]
* cattrs version:23.2.23
* Python version: 3.9
* Operating System:

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

Get intermittent failures with this lib.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```
line 355, in make_dict_structure_fn
    ian = a.alias
AttributeError: 'Attribute' object has no attribute 'alias'

**Comment by Tinche** [comment]
Hello,

Looks like your version of _attrs_ is too old. Which version are you using?

That shouldn't be possible since cattrs 23.2 depends on attrs >= 23.1.0, which should support aliases.

**Comment by Raghuveer123** [comment]
Yes…This issue can be closed now..This got fixed by matching the versions cattrs 23.2 and attrs >=23.1.0..

**Comment by Raghuveer123** [comment]
Closed
